administrator/language/en-GB/plg_task_checkfiles.ini

```
@@ -5,12 +5,14 @@
 
 PLG_TASK_CHECK_FILES="Task - Check Files"
 PLG_TASK_CHECK_FILES_LABEL_DIMENSION_LIMIT="Limit (px)"
+PLG_TASK_CHECK_FILES_LABEL_DIMENSION_NUMIMAGES="Number of Images"
```

```suggestion
PLG_TASK_CHECK_FILES_LABEL_DIMENSION_NUMIMAGES="Maximum Images"
```
Probably change the key to not include `DIMENSION`.

---

Thx, improved the strings.